[
  {
    "class": "django.core.exceptions.ValidationError",
    "message": "['This field is invalid as its value is forbidden']",
    "stack": [
      "src/oscar/core/validators.py:113"
    ]
  }
]